app/Model/AppModel.php

```diff
@@ -2180,6 +2193,18 @@ private function __addIndex($table, $field, $length = null, $unique = false)
         return $additionResult;
     }
 
+    private function __checkIndexExists($table, $column_name, $is_unique = false): bool
+    {
+        $query = sprintf(
```

Comment:
This could be dangerous in the future. Not an issue in this context, but we need to make sure that column_name and table name are not arbitrarily entered ever (unless we move the function to use prepared statements)